### DISCUSSION TITLE
[Inquiry] Curation of Datasets

### DISCUSSION THREAD

**Discussion by ryan1288** [shukkkur/VolleyVision#16] [Q&A]
For your labeled dataset, could I please ask if you did any manual labeling? Or did you only use open-sourced dataset? I'm interested in applying ML to volleyball but find data the primary restriction in creating well-performing open-sourced models. If so, what did you use?

**Comment by shukkkur** [comment]
So, for the [datasets](https://github.com/shukkkur/VolleyVision?tab=readme-ov-file#-datasets), I only labelled around 5k images for [volleyball](https://universe.roboflow.com/shukur-sabzaliev1/volleyball_v2). The rest of the images and datasets I collected across the internet, my goal was to combine anything there was for volleyball.

**Reply by ryan1288** [reply]
Thank you for your response! To clarify, you specifically only labeled for volleyball detection and sourced online datasets for other tasks. Is that correct? Thanks for clarifying

**Reply by ryan1288** [reply]
I assume you did the labeling from within Roboflow using their annotation tools?
